### PR TITLE
Added 'install' to build.clj

### PR DESCRIPTION
**Pull request by simon-brooke**
This allows me to add a dependency on your project to mine, and have it auto-resolved by Maven.

### CODE REVIEW

**Comment by simon-brooke** [comment]
H'mmm... don't merge this yet. I'm not getting your dependencies (e.g. bouncycastle, clj-http) automatically added to mine, so there's something more I need to do.

**Comment by simon-brooke** [comment]
OK, this works now and is ready to merge if you approve it.

**Review by jahfer — APPROVED** [review]
This looks great! I had been trying to figure out how to make `deps.edn` work to define the library as a dependency, thanks for figuring it out! 🎉

**Pull request merged**
